README.md

```diff
@@ -122,6 +122,18 @@ $ gcloud auth login
 You will need a project ID before you can run. Please navigate to https://console.developers.google.com and
 create one.
 
+
+## OpenStack nova client
+Install [pip](http://pip.readthedocs.org/en/latest/installing.html).
```

Comment:
*pip* is already a prerequisite for using PKB. Suggest following the example in the *Install AWS CLI* section:

> Make sure you have installed pip (see the section above).